.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+[STORY_LINK]
+
+## Why?
+
+Describe why you do this PR.
```

Comment:
Can we write "Describe the reason for creating this PR" Is it better or not?
```suggestion
Describe the reason for creating this **PR**.
```

Reply:
I would go for "To describe the reason for creating this PR"

Reply:
IMO, I'm fine with both actually. Then I'll leave the descision to @Kittinske15

Reply:
```suggestion
To describe the reason for creating this **PR**.
```

Reply:
test